memory_test.go

```diff
@@ -111,6 +116,27 @@ func TestParseMemoryStats(t *testing.T) {
 	}
 }
 
+func TestParseMemoryOomControl(t *testing.T) {
+	var (
+		c = &memoryController{}
+		m = &v1.MemoryOomControl{}
+		r = strings.NewReader(memoryOomControlData)
+	)
+	if err := c.parseOomControlStats(r, m); err != nil {
+		t.Fatal(err)
+	}
+	index := []uint64{
+		m.OomKillDisable,
+		m.UnderOom,
+		m.OomKill,
+	}
+	for i, v := range index {
```

Comment:
this loop looks weird, but not new in this PR, so LGTM

Reply:
ack